src/main/java/com/gooddata/sdk/common/collections/UriPageRequest.java

```diff
@@ -40,28 +41,28 @@ public UriPageRequest(final String pageUri) {
     /**
      * This is effectively no-op. Returns internal URI provided by REST API.
      *
-     * @param uriBuilder not used internally, can be null
+     * @param mutableUri not used internally, can be null
      * @return next page URI provided by REST API
      */
     @Override
-    public URI getPageUri(final UriComponentsBuilder uriBuilder) {
+    public URI getPageUri(final MutableUri mutableUri) {
```

Comment:
Why do you name arguments in this impl. different way than in ifc?

Reply:
I apparently forgot to rename it at some places... will fix that